starter-code/app.js

```diff
@@ -0,0 +1,43 @@
+
+const express = require('express');
+const hbs     = require('hbs');
+const app     = express();
+const path = require('path');
+
+
+app.set('view engine', 'hbs');
+app.set('views', __dirname + '/views');
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+app.get('/home', (req, res, next) => {
```

Comment:
It's better to have your home page as `app.get('/'` because this is the default page